charts/platform-factory/values.yaml

```diff
@@ -1,82 +1,99 @@
+prefix: "factory"
+# Override .Release.Name
 nameOverride: ""
+# Override all naming. Ignores `prefix`
 fullnameOverride: ""
+# Supported Values: `flux`
+# TODO: Support `argo`
+solution:
+  name: flux
+  interval: 10m
+  namespace: flux-system
 
-serviceAccount:
-  # Specifies whether a service account should be created
-  create: true
-  # Annotations to add to the service account
+global:
+  affinity: {}
   annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
-  name: ""
-
-podAnnotations: {}
-
-podSecurityContext: {}
-  # fsGroup: 2000
-
-securityContext: {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
-
-service:
-  type: ClusterIP
-  port: 80
-
-ingress:
-  enabled: false
-  className: ""
-  annotations: {}
-    # kubernetes.io/ingress.class: nginx
-    # kubernetes.io/tls-acme: "true"
-  hosts:
-    - host: chart-example.local
-      paths:
-        - path: /
-          pathType: ImplementationSpecific
-  tls: []
-  #  - secretName: chart-example-tls
-  #    hosts:
-  #      - chart-example.local
-
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
-
-autoscaling:
-  enabled: false
-  minReplicas: 1
-  maxReplicas: 100
-  targetCPUUtilizationPercentage: 80
-  # targetMemoryUtilizationPercentage: 80
-
-nodeSelector: {}
+  labels: {}
+  nodeSelector: {}
+  tolerations: []
+  istioEnabled: true
+  skyfjell:
+    validate:
+      api: true
 
-tolerations: []
+components:
+  # cert-manager Configuration
+  certManager:
+    # Enable cert-manager Resources
+    enabled: true
+    # Certificate `issuerRef``
+    issuerRef: true
+  istio:
+    enabled: true
+    namespace: "system-istio"
+    components:
+      gateway:
+        enabled: true
+        # Istio Ingress Gateway Namespace for Apps
+        namespace: system-istio-gateway
+        # For `app` label selector
+        name: ingress
+        # Selector - Defaults to `{ app: [name] }`
+        selector: {}
+  factories:
+    components:
+      privatebin:
+        enabled: false
+        name: "privatebin"
+        namespace: ""
+        host: ""
+        chart:
+          version: "0.17.0"
+          source:
+            kind: HelmRepository
+            # uses chart's included helm repository name
+            name: ""
+        annotations: {}
+        nodeSelector: {}
+        tolerations: []
+        affinity: {}
+        labels: {}
+        overrideValues: {}
+  # gitlab:
```

Comment:
Can uncomment these for the initial release prob. Let's release w/ at least pb/gitlab/jupyter running(if not accessible).